scripts/db_migrator.py

```diff
@@ -868,7 +868,8 @@ def common_migration_ops(self):
                 new_cfg = {**init_cfg, **curr_cfg}
                 self.configDB.set_entry(init_cfg_table, key, new_cfg)
 
-        self.migrate_copp_table()
+        if self.asic_type != "mellanox":
```

Comment:
It is not clear if there is any differences in the calls made to syncd/SAI. Can you explain why this check needs to be platform specific? Please also explain the difference in terms of SAI API calls?

Reply:
yeah, would like to know as well. Much of the coppmgr code is same for both the cases except handling dup entries, not clear what differs.

Reply:
This check is platform specific since it was brought to my attention that avoiding clearing the copp tables might cause issues on Broadcom platforms. I am not familiar with their implementation and from my point of view this can be platform agnostic as it was commented in the review. I guess the issue is with SAI handling overwrites or writing to create-only attributes.
The coppmgr code is platform agnostic as mentioned.
I will add a comment in the code according to @venkatmahalingam comment and in case you see that this change works fine with other vendors as well they can be added to the condition check. In case no vendor has any issues with that the condition check can be removed and then just delete calling migrate_copp_table function.

Reply:
I would have thought copporch listening to AppDb should never get update for new created entries and should never get create for updated entries, so not sure what scenario creates writing to create-only attributes?